cmd/exo/cmd/ssh.go

```diff
@@ -68,13 +99,17 @@ func getSSHInfos(name string) (*sshInfos, error) {
 	return &sshInfos{
 		sshKeys:  path.Join(configFolder, "instances", vm.ID, "id_rsa"),
 		userName: tempUser,
-		ip:       vm.IP(),
+		ip:       vmIP,
 		vmName:   vm.Name,
 		vmID:     vm.ID,
 	}, nil
 
 }
 
+func printSSHConnectSTR(infos *sshInfos) {
+	fmt.Printf("ssh -i %s %s@%s\n", infos.sshKeys, infos.userName, infos.ip)
```

Comment:
I'd love to see a check for the presence of the ssh key file.

Reply:
i had think about that this morning too. but forgotten to write it somewhere